docs/source-pytorch/fabric/api/fabric_methods.rst

```diff
@@ -127,25 +127,55 @@ This avoids excessive printing and logs when running on multiple devices/nodes.
 save
 ====
 
-Save contents to a checkpoint. Replaces all occurrences of ``torch.save(...)`` in your code. Fabric will take care of
-handling the saving part correctly, no matter if you are running a single device, multi-devices or multi-nodes.
+Save the state of objects to a checkpoint file.
+Replaces all occurrences of ``torch.save(...)`` in your code.
+Fabric will take care of handling the saving part correctly, no matter if you are running a single device, multi-devices or multi-nodes.
 
 .. code-block:: python
 
-    # Instead of `torch.save(...)`, call:
-    fabric.save(model.state_dict(), "path/to/checkpoint.ckpt")
+    # Define the state of your program/loop
+    state = {
+        "model1": model1,
+        "model2": model2,
+        "optimizer": optimizer,
+        "iteration": iteration,
+    }
+
+    # Instead of `torch.save(...)`
+    fabric.save("path/to/checkpoint.ckpt", state)
+
+You can pass in the model and optimizer objects directly, and Fabric will unwrap them and retrieve their *state-dict* automatically.
```

Comment:
Isn't this a "must" more than a "can"?

Reply:
I think "must" is too strong, but we want to recommend it in general because then the strategy can access the state in its own and efficient way. BUT, if you know what you are doing, nothing prevents you from calling state_dict yourself and pass the dicts instead of objects. This is totally fine as well. 

Ok, changed the formulation